### PR TITLE
php redis adapter

### PR DESCRIPTION
**Pull request by juliangut**
PHP Redis extension cache adapter

covers #73

### CODE REVIEW

**Comment by desarrolla2** [comment]
Hi @juliangut.

I just send a invitation to collaborate. Please merge the PR yourself

Regards!

File: src/Redis.php [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function get($key, $default = null)
+    {
+        $response = $this->client->get($this->keyToId($key));
+
+        return !empty($response) ? $this->unpack($response) : $default;
```
Comment (jasny) [inline src/Redis.php:98]: Check as `$response !== false`. Response may be an empty string or `0` if a different serializer is used.

**Review by jasny — CHANGES_REQUESTED** [review]

File: src/Redis.php (hunk 1 above)
Comment (juliangut) [reply src/Redis.php:98]: According to https://www.php.net/manual/en/function.empty.php `false`, `''` and `0` are considered empty, in all those cases $default will be returned

Comment (jasny) [reply src/Redis.php:98]: @juliangut `0` is a valid cached value. If `0` is cached it should be returned and not the default value.

**Pull request merged**